src/main/kotlin/no/nav/syfo/application/api/authentication/ApiFeature.kt

```diff
@@ -106,13 +116,27 @@ fun Application.installStatusPages() {
                 }
             }
 
-            val message = cause.message ?: "Unknown error"
-            call.respond(responseStatus, message)
-
             val callId = getCallId()
             val consumerId = getConsumerId()
-            log.error("Caught exception, callId=$callId, consumerId=$consumerId", cause)
-            throw cause
+            val errorMessage = "Caught exception, callId=$callId, consumerClientId=$consumerId"
+            when (cause) {
+                is ForbiddenAccessVeilederException -> {
+                    log.info(errorMessage, cause)
+                }
+                is IllegalArgumentException -> {
+                    log.warn(errorMessage, cause)
+                }
+                else -> {
+                    log.error(errorMessage, cause)
+                }
+            }
+            val message = cause.message ?: "Unknown error"
```

Comment:
Ref den andre PR'en din fra fredag (#244): Kanskje vi ikke burde cause.message som respons som default her? Da står vi litt friere til å logge nyttig kontekst uten at det flyter ut i responsen.

Dvs for exceptions som RuntimeException og andre exceptions som ikke håndteres for seg.